virttest/utils_test/libvirt.py

```diff
@@ -2567,9 +2567,8 @@ def set_vm_disk(vm, params, tmp_dir=None, test=None):
                         (src_disk_format, disk_format, blk_source, dist_img))
         else:
             # create another disk without convert
-            disk_cmd = "qemu-img create -f %s /mnt/%s 10M" % (src_disk_format,
-                                                              dist_img)
-
+            disk_cmd = "qemu-img create -f %s -o preallocation=full /mnt/%s 10M" % (src_disk_format,
```

Comment:
is there is special case that need this option : preallocation=full?
Better we keep it as pass in parameter,rather than hardcode his  option in  since there are quite numbers of cases dependent on this method.

Reply:
@chunfuwen this comes from https://github.com/avocado-framework/avocado-vt/pull/2914. The feature owner explained to me that this is to make sure a 'no space' error message is able to be displayed in logging file when 'qemu-img create'. Without it,  'qemu-img create' will not report an error and then vm starting will hang and no error message is displayed.